ldes-server-infra-mongo/mongo-repository/src/test/resources/features/ldesfragment/ldesfragment.feature

```diff
@@ -0,0 +1,12 @@
+Feature: LdesFragmentRepository
```

Comment:
Same question as with members, should we cover the other parts of the repo as well?

Reply:
created an issue for this: https://github.com/Informatievlaanderen/VSDS-LDESServer4J/issues/553